pkg/restore/restore.go

```diff
@@ -1649,12 +1691,26 @@ func resetMetadataAndStatus(obj *unstructured.Unstructured) (*unstructured.Unstr
 		}
 	}
 
-	// Never restore status
-	delete(obj.UnstructuredContent(), "status")
+	return obj, nil
+}
 
+func resetStatus(obj *unstructured.Unstructured) (*unstructured.Unstructured, error) {
```

Comment:
It's always nil, why return this error?

Reply:
This was part of the old code that I just splitted the `resetMetadataAndStatus` function.. 
But sure, I believe it's better to use the unstructured API :)